ska_dbi/__init__.py

```diff
@@ -2,6 +2,7 @@
 import ska_helpers
 
 from .DBI import *
+from .sqsh import Sqsh
```

Comment:
I think this doesn't need to be in the top-level API. Basically `mica` is the only user, right?